plugins/exporter.koplugin/main.lua

```diff
@@ -174,7 +174,7 @@ function Exporter:exportAllNotes()
 end
 
 --- Parse and export highlights from selected documents.
--- @tparam table files list of files as a table of {[file_path] = true}
+-- @tparam table files list of files 'file_path = true'
```

Comment:
Oh yeah, `{}` is probably a reference or something which it then can't find. For future reference, it might work with 
```
`{}`
```